search/search_quality/assessment_tool/results_view.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <cstddef>
+#include <vector>
+
+#include <QtWidgets/QListWidget>
+
+class QWidget;
+class ResultView;
+
+namespace search
+{
+class Result;
+}
+
+class ResultsView : private QListWidget
+{
+public:
+  explicit ResultsView(QWidget & parent);
+
+  void Add(search::Result const & result);
+
+  ResultView & Get(size_t i);
+  ResultView const & Get(size_t i) const;
+
+  size_t Size() const { return m_results.size(); }
```

Comment:
Short (one-line) methods may be defined in the header file.

https://google.github.io/styleguide/cppguide.html#Inline_Functions